src/platform/posix/posix.c

```diff
@@ -9,10 +9,10 @@
 #include <sof/schedule/ll_schedule.h>
 #include <sof/lib/agent.h>
 
-uint8_t posix_hostbox[MAILBOX_HOSTBOX_SIZE];
-uint8_t posix_dspbox[MAILBOX_DSPBOX_SIZE];
-uint8_t posix_stream[MAILBOX_STREAM_SIZE];
-uint8_t posix_trace[MAILBOX_TRACE_SIZE];
+uint8_t posix_hostbox[MAILBOX_HOSTBOX_SIZE] __aligned(8);
+uint8_t posix_dspbox[MAILBOX_DSPBOX_SIZE] __aligned(8);
+uint8_t posix_stream[MAILBOX_STREAM_SIZE] __aligned(8);
+uint8_t posix_trace[MAILBOX_TRACE_SIZE] __aligned(8);
```

Comment:
TBH I don't quite understand what this means. Do you mean, that when the fuzzer attempts to use objects, that must be aligned, they're copied into unaligned memory which breaks them? Why 8 bytes - do those objects contain 64-bit members?

Reply:
Yes that is exactly what happens. Right now I'm hitting 4 byte aligned structure access on unaligned addresses which UBSAN crashes on. See the comment in the PR for why I picked 8.

Reply:
Also if the decision is we need to have a fixed alignment, probably would IMHO look cleanest to define the arrays as uint64_t or uint32_t or whatever.  There's no reason for them to be bytes per se.

Reply:
The data objects being copied to/from the mailbox are 32bit aligned, so probably better to use uint32_t here.